### ISSUE TITLE
Only 6dof controller move out the security zone,There's no warning,the security fence was not displayed.

### ISSUE THREAD

**Issue by dingsing2021** [Igalia/wolvic#50]
## Configuration

Wolvic version:  0.9.3
Wolvic build ID: 

Hardware: huawei vr glass 6dof 

## Steps to Reproduce

1. enter the Wolvic, Go to the home page.
2.then the 6dof controller move out the security zone , the head position still in the security zone.

## Current Behavior

There's no warning that the controller is out of the security zone, the security fence was not displayed.

## Expected Behavior

The security fence should be displayed to remind the user that the security zone is out.

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by svillar** [comment]
HVR is using OpenXR, so maybe it's related to issue #26

**Comment by svillar** [comment]
OK, so I have just tested it again with the workaround for controllers for newer HVR runtimes and I see now what's going on. The security fence is indeed shown but very slightly. Actually is exactly what is described in issue #29. I'll check what's preventing us from showing it more prominently.

**Comment by svillar** [comment]
I've just found the problem. It isn't in the Wolvic sources but in either the GL stack of the device or the way the play area wireframe geometry is defined.

If we remove the calls to `glEnable(GL_CULL_FACE)` then the wireframe is shown, but we shouldn't have to do that because backface culling should basically always be used in order to have good performance.

I have filled [this issue](https://github.com/hms-ecosystem/OpenXR-SDK/issues/30) in the HMS ecosystem project. I'll keep this open just to track the issue.
